src/main/java/com/zerohunger/pdsmanagement/service/impl/StateManagementServiceImpl.java

```diff
@@ -84,4 +90,32 @@
 		log.info("Get Order Status Service Started !");
 		return Mono.just(requestStatusRepo.findOneByRequestId(requestId));
 	}
+
+	
+	public Object updateRation(String requestId, String stateName, Double quantityGranted, UpdateRation objectType) {
+		
+		Object object = null;
```

Comment:
Please try to avoid "null" as much as possible in code, use Optional wherever possible.